.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "Mochi/MochiMochi"]
 	path = Mochi/MochiMochi
 	url = https://github.com/georgeslabreche/MochiMochi
-	branch = orbitai
+	branch = factory_pattern
```

Comment:
@TanguySoto I'm going to put this back to the `orbitai` branch after everything is merged.